src/main/kotlin/io/gitlab/arturbosch/detekt/sonar/sensor/DetektConfiguration.kt

```diff
@@ -57,12 +57,12 @@ private fun tryFindDetektConfigurationFile(settings: Settings, baseDir: File): F
 		var configFile = File(path)
 
 		if (!configFile.exists() || configFile.endsWith(".yaml")) {
```

Comment:
Also, I do not understand this `.endsWith(".yaml")`.
On a `File`, [`endsWith`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/java.io.-file/ends-with.html) determines whether two files belong to the same root and end with the same components in the same order. There's no notion of extension, right?

Reply:
Hm I'm not sure. Can't find the kotlin sources for that, decompiling FilesKt does nothing ....
But I think your right, the `Path` object from java7 also works like that. Will change it to `toString.endsWith`. thx!